server/src/main/java/io/druid/server/ClientQuerySegmentWalker.java

```diff
@@ -84,25 +82,13 @@ public ClientQuerySegmentWalker(
     final FinalizeResultsQueryRunner<T> baseRunner = new FinalizeResultsQueryRunner<T>(
         toolChest.postMergeQueryDecoration(
             toolChest.mergeResults(
-                new MetricsEmittingQueryRunner<T>(
```

Comment:
losing that metric should be ok (when no chunking) as numbers reported in query/time and request/time were pretty much be the same.